vars/kibanaCoverage.groovy

```diff
@@ -0,0 +1,103 @@
+def bootMergeAndIngest(buildNum, buildUrl) {
+  kibanaPipeline.bash("""
+    source src/dev/ci_setup/setup_env.sh
+
+    # bootstrap from x-pack folder
+    cd x-pack
+    yarn kbn bootstrap --prefer-offline
+
+    # Return to project root
+    cd ..
+
+    . src/dev/code_coverage/shell_scripts/extract_archives.sh
+
+    . src/dev/code_coverage/shell_scripts/fix_html_reports_parallel.sh
+
+    . src/dev/code_coverage/shell_scripts/merge_jest_and_functional.sh
+
+    . src/dev/code_coverage/shell_scripts/copy_mocha_reports.sh
+
+    . src/dev/code_coverage/shell_scripts/ingest_coverage.sh ${buildNum} ${buildUrl}
+
+  """, "### Bootstrap shell and kibana env, merge and ingest code coverage")
+}
+
+def liveSitePrefix() {
+  return "gs://elastic-bekitzur-kibana-coverage-live/"
```

Comment:
does it make sense to merge this into the existing uploads?  not an opinionated question here, more trying to see if there's any differences

Reply:
Hey there!  What exactly are you referring to?  The historical data?

Reply:
As part of the bucket here - https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/45847/gcsObjects/